### DISCUSSION TITLE
Solution to importing albums from Google Takeout

### DISCUSSION THREAD

**Discussion by TheLastGimbus** [photoprism/photoprism#987] [Show and tell]
Hi there!

I created a [script for cleaning up Google Photos takeout](TheLastGimbus/GooglePhotosTakeoutHelper), and have been maintaining for a while.

Recently, I learned that you guys alse had this denatable pleause to integrate Google Takeout as import option: https://docs.photoprism.org/user-guide/use-cases/google/

> Note: Google Photos albums won't be imported yet as we're trying to find a way to deal with auto-generated albums users may not want to import.

@bitsondatadev made a nice solution to dealing with those albums: TheLastGimbus/GooglePhotosTakeoutHelper#36

Perhaps, this could help you somehow 👀

**Comment by graciousgrey** [comment]
Thank you for the hint, we will have a look as soon as we find time :)

**Reply by berot3** [reply]
Any news on this front? 🙂👀

**Reply by TheLastGimbus** [reply]
I recently re-wrote it to `v3` in Dart, and it's working even better and more stable, so you guys can cheick it out again :+1:

Also, you could tell me what's the best "album solution" for your Photoprism to import: https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/discussions/135
